### DISCUSSION TITLE
Editing RPU L0 metadata

### DISCUSSION THREAD

**Discussion by dadogggothigh** [quietvoid/dovi_tool#268] [General]
Hi all, 

is there a way to edit the L0 metadata of a RPU file with the dovi_tool? I looked at the examples and the editor's options, but couldn't find anything relative. And if it is not possible - does anyone know how this can be done?

Many thanks

**Comment by quietvoid** [comment]
Can you clarify what you're trying to achieve? L0 is only defined for XML metadata and some bits happen to end up in the RPU.

**Reply by dadogggothigh** [reply]
I'd like to change the target display settings. I could change the L6 metadata, but the mastering display in L0 remained the same as before (see below). In the RPU output below I'd like to change the RPU mastering display from 4000 nits to 1000 nits

  RPU mastering display: 0.0050/4000 nits
  RPU content light level (L1): MaxCLL: 4252.32 nits, MaxFALL: 421.19 nits
  L6 metadata: Mastering display: 0.0050/1000 nits. MaxCLL: 0 nits, MaxFALL: 0 nits
  L2 trims: 100 nits

**Reply by quietvoid** [reply]
You're looking for `min_pq` and `max_pq` in the edit config JSON.
They must be PQ codes in 12 bits values, so 1000 would be 3079.

**Reply by dadogggothigh** [reply]
great, thanks a lot
